src/service.ts

```diff
@@ -2,40 +2,66 @@ process.env.FORCE_COLOR = "0";
 
 import LRU from "nanolru";
 import path from "path";
-import prettier from "prettier";
-import { Options, format } from "prettier";
+import type { Options } from "prettier";
 
 const configCache = new LRU({ max: 20, maxAge: 60000 });
 
-async function resolveConfig(cwd: string, filepath: string): Promise<Options> {
+async function resolveConfig(
+  prettier: any,
+  cwd: string,
+  filepath: string
+): Promise<Options> {
   let v = configCache.get<string, Options>(cwd);
+
   if (!v) {
     v = await prettier.resolveConfig(filepath, {
       editorconfig: true,
       useCache: false,
     });
+
+    if (!v && typeof process.env.PRETTIERD_DEFAULT_CONFIG === "string") {
```

Comment:
Should we just rely on truthiness here? Or is there a use case for setting `PRETTIERD_DEFAULT_CONFIG` to an empty string?

```suggestion
    if (!v && process.env.PRETTIERD_DEFAULT_CONFIG) {
```

Reply:
This one should not cause a problem, because if we pass in an invalid string the prettier throws error. I was planning to add an `fs` check but it seems prettier already handled that.

Reply:
I know, but I think just relying on truthiness reads better than the use of `typeof`. What do you think?

Reply:
Done. Of course, I just misread it.